modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCachePartitionExchangeManager.java

```diff
@@ -1607,6 +1618,44 @@ private void processSinglePartitionRequest(ClusterNode node, GridDhtPartitionsSi
         }
     }
 
+    /**
+     * @param nodeId Node id.
+     * @param req Request.
+     */
+    private void processExchangeCheckRequest(UUID nodeId, PartitionsExchangeFinishedCheckRequest req) {
```

Comment:
`req` parameter is not used

Reply:
This is intentional. I wanted all process***Message methods to have similar signature.